examples/src/status-message-borsh.js

```diff
@@ -0,0 +1,41 @@
+import {NearBindgen, call, view, near} from "near-sdk-js";
+import * as borsh from 'borsh';
+
+const schema = {
+    struct: { records: {map: { key: 'string', value: 'string' }} }
+};
+
+function borshSerializeStatusMessage(statusMessage) {
+    return borsh.serialize(schema, statusMessage);
+}
+
+function borshDeserializeStatusMessage(value) {
+    return  borsh.deserialize(schema, value)
+}
+
+@NearBindgen({
+    serializer(value) {
+        return borshSerializeStatusMessage(value);
```

Comment:
if the method will simply call `borsh.serialize`, then maybe we could bring the call here? (same comment for deserialiser)

Reply:
updated!